inference-engine/src/vpu/myriad_plugin/CMakeLists.txt

```diff
@@ -37,9 +37,15 @@ endif()
 # sem_* POSIX functions must be overriden on OSX,
 # these functions come from "mvnc" target,
 # "mvnc" must be the first library in the link list
+set(LINK_LIBRARIES)
+list(APPEND LINK_LIBRARIES mvnc inference_engine inference_engine_legacy vpu_graph_transformer)
+# MyriadPlugin is not safe to be unloaded from memory at runtime
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    list(APPEND LINK_LIBRARIES "-Wl,-z,nodelete")
```

Comment:
it's not a static option I suppose. Can we use approach I suggested in the commend above?

Reply:
Done.